variablegroups/RPE-AKS-Common.json

```diff
@@ -1,59 +0,0 @@
-{
```

Comment:
is it worth keeping these so we can setup an RPE tenant env easily?

Reply:
I am not sure if this is being used and up to date . Especially didn't want to check in something which isn't working and tested . Does it make sense ?

Reply:
Added them back.